package.json

```diff
@@ -26,7 +26,9 @@
   "peerDependencies": {
     "@sentry/browser": ">=4.0.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@fullstorydev/browser": "^1.0.0"
```

Comment:
@patrick-fs I think we want `@fullstorydev/browser` as a peer dependency so that you don't have to include it in the bundle and so you don't end up having multiple versions of the package. See: https://nodejs.org/es/blog/npm/peer-dependencies/

Reply:
`@fullstorydev/browser` has been added as a peer dependancy.